### DISCUSSION TITLE
Notify the bumpVersion in the PR description for NuGet package manager

### DISCUSSION THREAD

**Discussion by gioce90** [renovatebot/renovate#32767] [Suggest an Idea]
### Tell us more.

Following this feature introduced time ago (#14348 ). Now is possible to use bumpVersion, but I noticed that the PR description doesn't contains this useful info. 

I thing that will be really useful a line who says:
`"This PR will bump the version of My.Package.Name from x.y.z --> [semVer new version, accordingly the configs]" `
or something like that

**Comment by rarkins** [comment]
Would you put it in the same table as other updates, or keep it separate?

We also need to consider that multiple files could be bumped in a single PR

**Reply by gioce90** [reply]
I think separate is the right choice.

And when there are multiple versions to be bumped... maybe a table is a good choice.
